.vscode/settings.json

```diff
@@ -3,5 +3,6 @@
   "editor.formatOnSave": true,
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "javascript.updateImportsOnFileMove.enabled": "always",
-  "typescript.updateImportsOnFileMove.enabled": "always"
+  "typescript.updateImportsOnFileMove.enabled": "always",
+  "i18n-ally.localesPaths": ["i18n", "i18n/locales"]
```

Comment:
I dont want us to add things like this without a discussion
can you explain more what this is used for?